assets/src/edit-story/components/errorBoundary/errorActions.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const Message = styled.p`
+  color: #fff;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 1.6;
+  padding: 20px;
+`;
+
+function ErrorActions() {
+  const reload = () => {
+    window.location.reload(true);
+  };
+
+  return (
+    <Message>
+      {__('Editor has crashed.', 'web-stories')}
+      <br />
+      {__("Try to reload if that doesn't help, wait for a fix.", 'web-stories')}
+      <br />
+      <button onClick={reload}>{__('Reload', 'web-stories')}</button>
+    </Message>
```

Comment:
Exact message TBD.

Reply:
@o-fernandez any suggestions?

Reply:
@merapi Can you do the following for now (untested):

```js
// TODO: Replace URL with correct one and pass error & errorInfo
const reportUrl = 'https://github.com/google/web-stories-wp/issues';

return (
  <Message>
    <p>{__('Something went wrong!', 'web-stories')}</p>
	<p>{__('The editor has crashed. Please try reloading the page and report the error using the buttons below.', 'web-stories')}</p>
	<p>{__('Apologies for the inconvenience.', 'web-stories')}</p>
	<p>
	  <button onClick={reload}>{__('Reload', 'web-stories')}</button>
	  <a href={reportUrl} target="_blank" rel="noopener">{__('Report Error', 'web-stories')}</a>
    </p>
  </Message>
);
```

We'd also need some styling here. The button & the link should look identical, be next to each other, and be styled like the other primary buttons in the editor.

Reply:
First pass on this:
![image](https://user-images.githubusercontent.com/21036224/85126473-d41a7e80-b22d-11ea-91c7-c474068d84e0.png)

Reply:
Nice!